### DISCUSSION TITLE
haplotype database using fastq

### DISCUSSION THREAD

**Discussion by ParvaSharma** [maize-genetics/phg_v2#171] [Q&A]
Hi
Do we have to use fasta file for haplotype database creation? I don't see an option to use fastq file to be used at this step. In PHGv1 we have an option for 'populate a db using WGS'.
Thanks

**Comment by lynnjo** [comment]
Yes, you must use fasta files.   In phgv2,  the database is populated with haplotypes created from assembly genomes  aligned against a reference.  We have intentionally moved away from WGS for haplotype creation.  High quality assemblies should provide better data for imputation and other analysis.
